src/pages/office/components/termination-code/termination-code.scss

```diff
@@ -0,0 +1,8 @@
+.mes-select {
+  .alert-wrapper div.alert-radio-group {
+    max-height: 800px;
+  }
+  .alert-radio-label {
+    white-space: normal!important;
```

Comment:
You don't need `!important` as you can target the right element without.
```
.mes-select {
  .alert-radio-label{
    white-space: normal;
  }
}
```
Also the current code would change the wrapping for all select boxes, which may not be intended. So you could add a another class to the `ion-select` like
```
[selectOptions]="{cssClass:'mes-select termination-code-select'}"
```
and then target just this select control
```
.mes-select.termination-code-select {
  .alert-radio-label {
    white-space: normal;
  }
}
```